storagenode/piecestore/endpoint.go

```diff
@@ -358,7 +359,13 @@ func (endpoint *Endpoint) Upload(stream pb.DRPCPiecestore_UploadStream) (err err
 			mon.IntVal("upload_failure_size_bytes").Observe(uploadSize)
 			mon.IntVal("upload_failure_duration_ns").Observe(uploadDuration)
 			mon.FloatVal("upload_failure_rate_bytes_per_sec").Observe(uploadRate)
-			endpoint.log.Error("upload failed", zap.Stringer("Piece ID", limit.PieceId), zap.Stringer("Satellite ID", limit.SatelliteId), zap.Stringer("Action", limit.Action), zap.Error(err), zap.Int64("Size", uploadSize), remoteAddrLogField)
+			if errors.Is(err, context.Canceled) {
```

Comment:
I also observe _context canceled_ for downloads. Do you want to also patch it?

Reply:
Done, thanks for the heads up! It didn't occur to me to check for that. I notice the CI is failing right now - on my end, testing just that module shows a pass, and (if I'm understanding the output correctly - I may not be, I'm not super-experienced with Go just yet) the changes to this section are passing on Jenkins' end too, but failing as a whole. I'll some time aside later to debug this properly, though a more experienced pair of eyes is always welcome!